docs/10-action-optimize-media.md

```diff
@@ -0,0 +1,30 @@
+# Add an action to optimize media
+
+To add a new action in your sonata admin.
+
+![optimize-action.png](img/optimize-action.png)
+
+## Implementation
+
+You could extend `Ekino\TinyPngSonataMediaBundle\Controller\MediaAdminController` or use directly the `Ekino\TinyPngSonataMediaBundle\OptimizeImageTrait` in your Controller:
+```php
+<?php
+
+namespace App\Controller;
+
+use Sonata\MediaBundle\Controller\MediaAdminController as BaseMediaAdminController;
+use Ekino\TinyPngSonataMediaBundle\OptimizeImageTrait;
+
+class MediaAdminController extends BaseMediaAdminController
+{
+    use OptimizeImageTrait;
+}
+```
+
+To enable the extension, you have to declare it into `sonata_admin.yml`:
```

Comment:
I'd move it first, because this is the required config, extending the controller or using the trait is optional